gluonnlp/model/cache.py

```diff
@@ -70,6 +70,31 @@ def __init__(self, pretrained_lm_model, vocab_size, window, theta, lambdas, **kw
         with self.name_scope():
             self._pretrained_lm_model = pretrained_lm_model
 
+    def save_params(self, filename):
+        """Save parameters to file.
+
+        filename : str
+            Path to file.
+        """
+        self._children[list(self._children.keys())[0]].save_params(filename)
```

Comment:
self._pretrained_lm_model.save_params?

Reply:
both way should work. I made the change anyway